src/main/java/com/diffblue/deeptestutils/Reflector.java

```diff
@@ -449,7 +464,11 @@ private static void makePublic(final CtClass c) {
           .get(newClassName + "_implementation"));
       }
     } else {
-      return getInstance(Class.forName(className));
+      try {
+        return getInstance(Class.forName(className));
```

Comment:
where is `ExceptionInInitializerError` coming from?

Reply:
This is thrown in the case of an Exception in the static initializer. Interestingly this seems to happen on the `forName` already and not when instantiating the object type for the first time. Therefore it has to be caught here. This error contains the cause, to keep things simply I extract the cause and wrap it into an `InvocationTargetException` which is expected in the generated test.

Reply:
This is probably worth a comment in the code.